### DISCUSSION TITLE
iPhone-15 with ios17 never launches (simulator)

### DISCUSSION THREAD

**Discussion by mzdm** [codemagic-ci-cd/codemagic-docs#2190] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

`xcode: 15.0`

readonly DEVICE_NAME=Flutter-iPhone
readonly DEVICE=com.apple.CoreSimulator.SimDeviceType.iPhone-15
readonly OS=com.apple.CoreSimulator.SimRuntime.iOS-17-0

`DEVICE_ID=$(xcrun simctl create "$DEVICE_NAME" "$DEVICE" "$OS") && xcrun simctl boot "$DEVICE_ID" && open -a Simulator --args -CurrentDeviceUDID "$DEVICE_ID"`

This device type never launches or loads for very long time. Older versions fine. It is needed for integration tests.

### Expected results

Should load instantly

### Actual results

Simulator never loads

### Build id (optional)

654ce749d110cf5434f610b5

**Comment by icarusdust** [comment]
Hi, I just tested it and it works as expected. Try my steps below and let me know if it is still not working for you:

1. Remote access to a builder machine to check it closer.
2. Shutdown all the existing simulators by running `xcrun simctl shutdown all` in the terminal
3. Create the simulator with iPhone 15 ( iOS 17). You can run `xcrun xctrace list devices` to find a list of all the available devices with their iOS versions. To create one, I ran the following command:
`DEVICE=$(xcrun simctl create test-device com.apple.CoreSimulator.SimDeviceType.iPhone-11 com.apple.CoreSimulator.SimRuntime.iOS-15-0)`
4. Then you can boot it by running `xcrun simctl boot $DEVICE`, I opened the simulator by running `open -a Simulator --args -CurrentDeviceUDID $DEVICE_ID`

**Reply by mzdm** [reply]
I see `iPhone-11` in your command when creating, try `iPhone-15` and also ios 17

**Reply by icarusdust** [reply]
@mzdm it was supposed to be like `DEVICE=$(xcrun simctl create test-device com.apple.CoreSimulator.SimDeviceType.iPhone-15 com.apple.CoreSimulator.SimRuntime.iOS-17-0)`. I just tested multiple options, and pasted the incorrect one here. So, in short, it works with iPhone-15 and iOS 17

<img width="755" alt="Screenshot 2023-11-28 at 10 35 14" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/43378371/2c60f6ee-5248-4c91-af78-e675dee0d75e">

**Reply by mzdm** [reply]
Ah ok, thanks for trying. I will try it once more

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion due to a lack of activity. You can open the discussion again by replying to this message or opening a new discussion.
